URA-695/run_sentieon-tnbam.py

```
@@ -0,0 +1,102 @@
+import dxpy as dx
+import json
+
+def get_sample_bam_file(sample):
```

Ruff (0.8.0)

Missing return type annotation for public function `get_sample_bam_file`

(ANN201)